.github/workflows/maven.yml

```diff
@@ -4,14 +4,15 @@ on: [push]
 
 jobs:
   build:
-
-    runs-on: ubuntu-latest
```

Comment:
Porque não rodar na última versão? Ou, pelo menos, na ultima LTS?

Reply:
É sempre importante fixar a versão das bibliotecas e afins, para garantir que o build com aquelas informações não irão quebrar e você terá sempre o mesmo teste executando. Se você deixar ele alterar a versão sozinho, pode ser que o pipeline quebre, sem você ter feito nenhuma alteração nele, o que não é legal e também não é uma boa prática para entregas continuas.

Reply:
Eu ainda prefiro estar sempre na versão mais nova e corrigir uma eventual quebra, porque isso me força a estar sempre "na crista da onda". 
Mas, mesmo que não fosse assim, já existe o ubuntu 20.04 que é lts.

Reply:
Tá, ai acho que pode ficar a seu critério, se quiser posso mudar para a última versão. Estou preparando um PR para publicar também no github packages, ai posso alterar nele. Pode ser ?

Reply:
Pode ser sim! A ideia do github packages tb é muito boa, pode mandar que vamos botar pra cima rapidinho!
Abs